Mergin/utils.py

```diff
@@ -1679,3 +1683,33 @@ def is_inside(proj_dir: str, to_check: str) -> bool:
         return True
     except ValueError:
         return False
+
+
+def val_to_string(val: Any) -> str:
```

Comment:
I believe `val` in this case is Qt's [QVariant](https://doc.qt.io/qt-6/qvariant.html) 
```suggestion
def qvariant_to_string(val: Any) -> str:
```
otherwise, if it can be anything, you can go with simple

```suggestion
def to_string(val: Any) -> str:
```